arch/none/host/src/arch_main.c

```diff
@@ -5,33 +5,111 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+#include <glib.h>
```

Comment:
Do not change these files in arch/none/host, Can you add a new arch e.g. arch/none/host-scmi-server ?